apps/fabric-website/src/components/Nav/Nav.tsx

```diff
@@ -184,30 +180,14 @@ export class Nav extends React.Component<INavProps, INavState> {
       const key = `${page.title}-${sectionIndex}`;
       return (
         <li key={key} className={css(styles.section, hasActiveChild(page) && styles.hasActiveChild)}>
-          <CollapsibleSection
-            collapsed={!(this.state.collapsedSections[key] || hasActiveChild(page))}
-            title={{
-              text: page.title,
-              onClick: () => this._handleSectionClick(key)
-            }}
-          >
+          <CollapsibleSection defaultCollapsed={!hasActiveChild(page)} title={page.title}>
```

Comment:
Do you know if the Nav is destroyed and re-created when the active page changes? If so, this logic should work. If not, it probably won't work (provided CollapsibleSection is properly implementing controlled/uncontrolled behavior)--if the same CollapsibleSection instance is reused across renders, updates to the `defaultCollapsed` prop *should* be ignored.

Reply:
I'm pretty sure it is destroyed when the page changes. From my testing this works fine now, when it didn't before some of the updates Jason has made to `CollapsibleSection`.

Reply:
Feel free to test yourself as well: http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/9078/merge/fabric-website/dist/index.html?devhost#/controls/web/button

Reply:
Sounds good then!